src/EnergyPlus/FluidProperties.cc

```diff
@@ -4842,7 +4822,7 @@ namespace FluidProperties {
 
         // PURPOSE OF THIS FUNCTION:
         // This function checks on an input fluid property to make sure it is valid.
-        auto &df = state.dataFluidProps;
+        auto const &df = state.dataFluidProps;
```

Comment:
[src/EnergyPlus/FluidProperties.cc:4845]:(style),[constVariable],Variable 'df' can be declared as reference to const